Kalenteri/Backend/Services/OrderService.cs

```diff
@@ -14,12 +14,19 @@ public static class OrderServices
     private static readonly string? DatabaseId = Configuration["ConnectionStrings:DatabaseId"];
     private static readonly string? ContainerId = Configuration["ConnectionStrings:ContainerId"];
     
-    public static Order GetData(string identifier)
+    public static Order? GetData(string identifier)
     {
         Container container = CreateContainer();
-        return container.GetItemLinqQueryable<Order>(true)
-            .Where(o => o.Identifier == identifier)
-            .ToList().First();
+        try
+        {
+            return container.GetItemLinqQueryable<Order>(true)
+                .Where(o => o.Identifier == identifier)
+                .ToList().FirstOrDefault();
+        }
+        catch(InvalidOperationException e)
```

Comment:
Tarvitaanko tätä `try-catch`iä enää? Mun mielestä ei ja jos niin ainakin yhden warningin saa pois kun ottaa käyttämättömän muuttujan `e` pois 😉 

```suggestion
        catch(InvalidOperationException)
```

Reply:
Totta. Eihän tuota nyt tarvii mihinkään jos toi firstOrDefault palauttaa nullin.